lahja/endpoint.py

```diff
@@ -109,17 +254,23 @@ class Endpoint:
     _internal_queue: asyncio.Queue
     _internal_loop_running: asyncio.Event
 
-    _loop: asyncio.AbstractEventLoop
+    _server_running: asyncio.Event
```

Comment:
This seems prudent to have as a public API since it is likely a common thing to want to wait until the endpoint is active...